ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -232,6 +232,9 @@ measurement:
   voltage: "Electrical potential difference between two reference points."
   voltageratio: "The ratio of the Primary voltage of the transformer to the secondary voltage."
   volume: "The quantity of three-dimensional space contained by a closed surface."
+
+  #New measurement
+  flow: "Movement of water or gas outside container."
```

Comment:
Flowrate covers this if its measured in gallons per minute, liters per second, etc.

Reply:
i see. i'll remove this now